python/ray/data/impl/pandas_block.py

```diff
@@ -164,6 +174,73 @@ def _empty_table() -> "pandas.DataFrame":
     def _sample(self, n_samples: int, key: "SortKeyT") -> "pandas.DataFrame":
         return self._table[[k[0] for k in key]].sample(n_samples, ignore_index=True)
 
+    def count(self, on: KeyFn, ignore_nulls: bool) -> Optional[U]:
```

Comment:
I believe that the null handling in `_apply_agg()` precluded using it for `count()`, since `count()` is supposed to count the number of non-null values, but I'll try it out again to double-check.